mobile/lib/features/wallet/wallet_screen.dart

```diff
@@ -40,16 +37,41 @@ class WalletScreen extends StatelessWidget {
             children: [
               const Balance(),
               const SizedBox(height: 10.0),
-              if (walletChangeNotifier.lightning().sats == 0)
-                Container(
+              Container(
                   margin: const EdgeInsets.only(left: 4, right: 4),
-                  child: ElevatedButton(
-                    onPressed: () {
-                      context.go(OnboardingScreen.route);
-                    },
-                    child: const Text("Fund Wallet"),
-                  ),
-                ),
+                  child: Row(children: [
+                    Expanded(
+                      child: ElevatedButton(
+                        style: balanceButtonStyle,
+                        // Additionally checking the Lightning balance here, as when
+                        // hot reloading the app the channel info appears to be
+                        // unknown.
+                        onPressed: () => context.go(
+                            (hasChannel || walletChangeNotifier.lightning().sats > 0)
+                                ? ReceiveScreen.route
+                                : OnboardingScreen.route),
+                        child: const Text('Receive'),
+                      ),
+                    ),
+                    const SizedBox(width: 10.0),
+                    Expanded(
+                      child: ElevatedButton(
+                        style: balanceButtonStyle,
+                        onPressed: () {
+                          // TODO: Take me to the Lightning-USDp swap screen.
+                        },
```

Comment:
Do we want this done before merge?

Reply:
I think that will just route to the screen in your ticket: https://github.com/get10101/10101/issues/1676.